.github/CODEOWNERS

```diff
@@ -1115,5 +1115,5 @@
 #PRLabel: %CodeGen %Client
 /eng/scripts/typespec/                                             @JoshLove-msft @m-nash @jorgerangel-msft @jsquire
 
-# PRLabel: Azure Impact Reporting
+# PRLabel: %impactreporting - Azure Impact Reporting
```

Comment:
Labels cannot be arbitrarily created and should be coordinated with @ronniegeraghty.

Reply:
`Impact Reporting` has been approved as the service label and it has been pushed to all SDK repos.